.github/workflows/test-command.yml

```diff
@@ -251,7 +251,8 @@ jobs:
 
     # Run the terratest integration tests
     - name: "Test `examples/complete` with terratest on GitHub"
-      if: ${{ contains(github.event.client_payload.github.payload.repository.name, 'terraform-github-') }}
+      # Add terraform-null-label explicitly, as the other terraform-null- repos do not yet have tests
+      if: ${{ contains(github.event.client_payload.github.payload.repository.name, 'terraform-github-') || contains(github.event.client_payload.github.payload.repository.name, 'terraform-null-label') }}
```

Comment:
We want to create a new test for modules whose primary provider is `null`.

![image](https://user-images.githubusercontent.com/52489/91494380-99584680-e86d-11ea-97f8-9dfe45daddde.png)

> Test `examples/complete` with terratest on GitHub

"on GitHub" refers to running the tests against github using the github provider.